OrbitCore/UtilsTest.cpp

```diff
@@ -0,0 +1,17 @@
+#include <gtest/gtest.h>
+
+#include "Utils.h"
+
+TEST(Utils, TestEllipsis) {
+  EXPECT_EQ(ShortenStringWithEllipsis("17 char long text", 18), "17 char long text");
+  EXPECT_EQ(ShortenStringWithEllipsis("17 char long text", 17), "17 char long text");
+  EXPECT_EQ(ShortenStringWithEllipsis("17 char long text", 7), "17...xt");
+  EXPECT_EQ(ShortenStringWithEllipsis("17 char long text", 6), "1...xt");
```

Comment:
Do we want the ellipsis in the middle? I find it often undesirable as it takes away characters from the prefix and gives them to a suffix that is unreadable anyway after the ellipsis has been added.
Either way, in cases like this line I think that the tie should be broken in favor of the prefix, and the result should for example be "17...t".

Reply:
I tried to be consistent with the way QT is doing it in e.g. the tables. But you're right, the tie should prefer the prefix (which is also the case in QT), I'll fix that.

Reply:
Changed the behavior to cut the postfix before the prefix